Source/WebCore/rendering/RenderTextControlMultiLine.cpp

```diff
@@ -75,7 +75,7 @@ float RenderTextControlMultiLine::getAverageCharWidth()
     // Since Lucida Grande is the default font, we want this to match the width
     // of Courier New, the default font for textareas in IE, Firefox and Safari Win.
     // 1229 is the avgCharWidth value in the OS/2 table for Courier New.
-    if (style().fontCascade().firstFamily() == "Lucida Grande")
+    if (style().fontCascade().firstFamily() == "Lucida Grande"_s)
```

Comment:
This code is super-obsolete. Lucida Grande hasn’t been the default font for anything for a long, long time.